### PR TITLE
chore: improve experience for when lnd is slow to respond

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

**Review by vindard — APPROVED** [review]
Is there any place that we run `getBalance` from where it's ok to be slow?

I'm wondering if there's somewhere that we can still expose the errors if any happen.

File: src/app/wallets/get-balance-for-wallet.ts [hunk 1]
```diff
@@ -11,7 +11,12 @@ export const getBalanceForWallet = async ({
   lock?: DistributedLock
   logger: Logger
 }): Promise<Satoshis | ApplicationError> => {
-  const [, updatePaymentsResult] = await Promise.all([
+  // FIXME(nicolas, arvin): lnd take some time long to respond for updatePendingInvoice
+  // so removing the await will enable a balance that will return a lot more quickly
+  // the real fix for that is to implement https://github.com/GaloyMoney/galoy/issues/604
+  // so that there is no longer a dependency between getBalanceForWallet and updatePendingInvoices
+  // const [, updatePaymentsResult] =
```
Comment (dolcalmi) [inline src/app/wallets/get-balance-for-wallet.ts:18]: please remove

File: src/app/wallets/get-balance-for-wallet.ts [hunk 2]
```diff
@@ -23,7 +28,7 @@ export const getBalanceForWallet = async ({
       logger,
     }),
   ])
-  if (updatePaymentsResult instanceof Error) throw updatePaymentsResult
+  // if (updatePaymentsResult instanceof Error) throw updatePaymentsResult
```
Comment (dolcalmi) [inline src/app/wallets/get-balance-for-wallet.ts:31]: please remove

**Review by dolcalmi — CHANGES_REQUESTED** [review]

**Comment by nicolasburtey** [comment]
maybe a lot of the slow issues was related to the payment issues related to route hint? we may want to wait and see whether lnd is still slow to respond

**Comment by nicolasburtey** [comment]
will close this for now. we'll see if the fix we did last week + @vindard PR on payment cleanup will resolve this.

**Pull request closed without merging**